doc/_admin-guide/190_The_syslog-ng_manual_pages/001_dqtool_manual.md

```diff
@@ -122,7 +123,7 @@ Relocate every queue file:
 bin/dqtool relocate --new_path /tmp/dq --persist var/syslog-ng.persist --all
```

Comment:
please use the full path everywhere
/opt/syslog-ng/bin/dqtool
etc.
page generation will take care of the correct installation path replacement based on the "template" /opt/syslog-ng/ value